lug-helper.sh

```diff
@@ -2083,14 +2118,19 @@ rm_shaders() {
     fi
 
     # Loop through all possible shader directories
-    for appdata_dir in "$shaders_dir"/*; do
-        if [ -d "$appdata_dir/$shaders_subdir" ]; then
-            # If a shaders directory is found, delete it
-            if message question "The following directory will be deleted:\n\n$appdata_dir/$shaders_subdir\n\nDo you want to proceed?"; then
-                debug_print continue "Deleting $appdata_dir/$shaders_subdir..."
-                rm -r "${appdata_dir:?}/$shaders_subdir"
+    for appdata_dir in "$wine_prefix/$appdata_path"/*; do
```

Comment:
A question on this: I originally had `$wine_prefix/$appdata_path` set as a `$shaders_dir` variable in the `get_dirs()` function. Let me explain my original logic to see if it makes any sense to you or not. I just want to double check that we're improving the maintainability and readability of the code with this change.

My original thinking was that if CIG ever decided to switch around the directories on us, the `get_dirs` function alongside the other path declarations seemed like the obvious place to look first to update this. It also seemed like it would be easily discoverable in a search by being named `shaders_dir`. With this change, it's just the actual path right here in the shaders function. Also seems to be obvious and potentially easier to read as long as the reader notices the comments pointing out that this is shaders path. So given all that, what are your thoughts?
- What's the likelihood of CIG changing this path on us? Probably low at this point, but they've done it before right?
- If they did change it again, what's the easiest, quickest, most obvious place for us to change and update this path?
- If you were new to this code, which way do you think is easier to read and understand what's happening?

Reply:
sounds valid to me, my reasoning around the naming back then had been considering that there's now also at least one configuration file. changing it around to be consistent to the intention of get_dirs

Reply:
Which configuration file do you mean? If it makes sense to keep it the way you had it, we certainly can. Just wanted to make sure we're being intentional about the things I mentioned, you know?

Reply:
as example AppData/Local/Star Citizen/sc-alpha-3.23.1_cc9wt_0/GraphicsSettings/GraphicsSettings.json lives in there at present. it could perhaps warrant a more generic name than shaders_dir however I would think that if somebody was searching the code "shaders" is the word they would be expecting to find